ansible_ai_connect/users/views.py

```diff
@@ -108,6 +108,51 @@ def retrieve(self, request, *args, **kwargs):
         return Response(user_data)
 
 
+class markdownCurrentUserView(RetrieveAPIView):
```

Comment:
Would you rename this to `MarkdownCurrentUserView`?
Pyhon classnames usually start with an uppercase letter.